packages/augur-ui/src/modules/markets/reducers/new-market.ts

```diff
@@ -78,22 +78,25 @@ export default function(newMarket: NewMarket = DEFAULT_STATE, { type, data }: Ba
         type,
         orderEstimate,
         outcome,
-        outcomeName
+        outcomeName,
       } = orderToAdd;
       const existingOrders = newMarket.orderBook[outcome] || [];
+
       let orderAdded = false;
-      const updatedOrders = existingOrders.reduce((Orders: Array<LiquidityOrder>, order) => {
-        const orderInfo = Object.assign({}, order);
+
+      const updatedOrders = existingOrders.map((order: LiquidityOrder) => {
+          const orderInfo = Object.assign({}, order);
         if (createBigNumber(order.price).eq(createBigNumber(price)) && order.type === type) {
-          orderInfo.quantity = createBigNumber(order.quantity).plus(createBigNumber(quantity)).toNumber();
+          orderInfo.quantity = createBigNumber(order.quantity).plus(createBigNumber(quantity)).toString();
+          orderInfo.shares = createBigNumber(order.quantity).plus(createBigNumber(quantity)).toString();
           orderInfo.orderEstimate = createBigNumber(order.orderEstimate).plus(
             createBigNumber(orderEstimate.replace(" DAI", ""))
```

Comment:
not a fan of this logic, shouldn't be removing a string in order to treat something like a number.  need to just add `totalCost` to order that is used in new Market reducer and not parse out order estimate.